docs/intro/guides/drizzle/nextjs.rst

```diff
@@ -0,0 +1,271 @@
+        $ npx drizzle-kit pull
+
+
+.. edb:split-section::
+
+    This command will create a schema file based on your Gel database. The file will typically look something like this:
+
+    .. code-block:: typescript
+        :caption: drizzle/schema.ts
+        :class: collapsible
+
+        import { gelTable, uniqueIndex, uuid, smallint, text, timestamp, relations } from "drizzle-orm/gel-core"
+        import { sql } from "drizzle-orm"
+
+        export const books = gelTable("Book", {
+          id: uuid().default(sql`uuid_generate_v4()`).primaryKey().notNull(),
+          title: text().notNull(),
+          author: text(),
+          year: smallint(),
+          genre: text(),
+          read_date: timestamp(),
+        }, (table) => [
+          uniqueIndex("books_pkey").using("btree", table.id.asc().nullsLast().op("uuid_ops")),
+        ]);
+
+        export const notes = gelTable("Note", {
+          id: uuid().default(sql`uuid_generate_v4()`).primaryKey().notNull(),
+          text: text().notNull(),
+          created_at: timestamp().default(sql`datetime_current()`),
+          book_id: uuid().notNull(),
+        }, (table) => [
+          uniqueIndex("notes_pkey").using("btree", table.id.asc().nullsLast().op("uuid_ops")),
+        ]);
+
+
+Keep Drizzle in sync with Gel
+=============================
+
+.. edb:split-section::
+
+    To keep your Drizzle schema in sync with your Gel schema, add a hook to your ``gel.toml`` file. This hook will automatically run ``drizzle-kit pull`` after each migration:
+
+    .. code-block:: toml
+        :caption: gel.toml
+
+        [hooks]
+        after_migration_apply = [
+          "npx drizzle-kit pull"
+        ]
+
+
+
+With this hook in place, your Drizzle schema will automatically update whenever you apply Gel migrations.
+
+
+Create a database client
+========================
+
+.. edb:split-section::
+
+    Now, let's create a database client that you can use throughout your application:
+
+    .. code-block:: typescript
+        :caption: src/db/index.ts
+
+        import { drizzle } from 'drizzle-orm/gel';
+        import { createClient } from 'gel-js';
+        import * as schema from '@/drizzle/schema';
+        import * as relations from '@/drizzle/relations';
+
+        // Import our schema
+        import * as schema from './schema';
+
+        // Initialize Gel client
+        const gelClient = createClient();
+
+        // Create Drizzle instance
+        export const db = drizzle({
+          client: gelClient, schema: {
+            ...schema,
+            ...relations
+          },
+        });
+
+        // Helper types for use in our application
+        export type Book = typeof schema.book.$inferSelect;
+        export type NewBook = typeof schema.book.$inferInsert;
+
+        export type Note = typeof schema.note.$inferSelect;
+        export type NewNote = typeof schema.note.$inferInsert;
+
+
+Perform database operations with Drizzle
+========================================
```

Comment:
Maybe we can link out to their [Querying](https://orm.drizzle.team/docs/rqb) docs here? Or add a direct link to each split section pointing at the relevant doc page? I guess we don't really want to be an index of their documentation, so probably a single link at the beginning with a short paragraph?